2_pvc_destination_gen/README.md

```diff
@@ -1,6 +1,32 @@
 # Stage 2: pvc-destination-gen
 
-This playbook takes the input from [Stage 1](../1_pvc_data_gen) and creates PVC OpenShift resources in destination cluster
+This playbook takes the input from [Stage 1](../1_pvc_data_gen) and creates PVC OpenShift resources in destination cluster.
+
+## Sub-Stages
+
+`Stage 2` is divided into two sub-stages `Stage 2 (a)` and `Stage 2 (b)`. 
+
+### Stage 2 (a)
```

Comment:
Is this new step optional, or is it required?

Reply:
@djwhatle It can be safely skipped using `--skip-tags adjust_pvs` It doesn't affect the original Stage 2 flow. But, it is required if users expect their PVs to be automatically adjusted by `pvc-migrate`